charts/wildfly/README.md

```diff
@@ -194,3 +194,8 @@ If the Helm chart is only used to build the application image, you can skip the
 | `deploy.tls.enabled` | Enables the creation of a secure service to access the application. If `true`, WildFly must be configured to enable HTTPS | `false`| |
 | `deploy.volumes` | Freeform `volumes` items| - | [Kubernetes Documentation](https://kubernetes.io/docs/concepts/storage/volumes/) |
 | `deploy.volumeMounts` | Freeform `volumeMounts` items| - | [Kubernetes Documentation](https://kubernetes.io/docs/concepts/storage/volumes/) |
+| `deploy.ingress.enabled` | Determines if a `Ingress` configuration should be created | `false` | Allows clients outside of Kubernetes to access your application |
+| `deploy.ingress.className` | Configure the `ingressClassName` is the name of the IngressClass cluster resource. The associated IngressClass defines which controller will implement the resource. If not set the default `ingressClass` is used.| |[Kubernetes Documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class). |
+| `deploy.ingress.host` | `host` is an alias/DNS that points to the service.  | | [Kubernete Documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource) |
```

Comment:
description does not look accurate... the host is used to determine where the inbound traffic is routed.